### DISCUSSION TITLE
I need help

### DISCUSSION THREAD

**Discussion by mobeicanyue** [giscus/giscus#1338] [Q&A] [answered]
If my giscuss app be copy to another people's blog, could he use my discussion?
I found out my discussion is used by another people now.
He create a blog, and use my giscus

**Comment by laymonage** [comment, marked as answer]
You can restrict the domains that can load giscus with your repository's discussions by adding a giscus.json to your repository.

https://github.com/giscus/giscus/blob/main/ADVANCED-USAGE.md#origins
